tools/js-sdk-release-tools/docs/hlc.md

```diff
@@ -0,0 +1,43 @@
+# Azure Track2 JS SDK Tools
+This package contains 3 tools used for azure track2 mgmt js sdk:
+1. Use Track2 Codegen to generate codes by inputting swagger, and then run `rush update & rush build`.
+2. Use Track2 Codegen to generate codes in swagger pipeline.
+3. Generate changelog and bump version for Azure Track2 JS SDK.
+
+## Install
+```shell script
+npm install -g @azure-tools/js-sdk-release-tools
+```
+
+## Tools
+
+### Use Track2 Codegen to generate codes by inputting swagger
+You can use this command to generate codes, and then run rush update, rush build, and pack the generated codes.
+```shell script
+cd azure-sdk-for-js
+hlc-code-gen --use=@autorest/typescript@6.0.0-beta.5 --readme=../azure-rest-api-specs/specification/iotspaces/resource-manager/readme.md
+```
+hint: you can change the `readme` to yours.
+
+### Use Track2 Codegen to generate codes in swagger pipeline
+This command is only used in swagger pipeline.
+```shell script
+cd azure-sdk-for-js
+hlc-code-ge-for-pipeline --inputJsonPath=a.json --outputJsonPath=b.json --use=@autorest/typescript@6.0.0-beta.5
```

Comment:
Maybe add a note here to tell customers find the latest code generator release version ?